### PR TITLE
Change probe and integral interface

### PR DESCRIPTION
**Pull request by benflexcompute**

### CODE REVIEW

File: tests/simulation/translator/ref/Flow360_CHT_three_cylinders.json [hunk 1]
```diff
@@ -198,5 +198,6 @@
             "thermalConductivity": 0.0239367,
             "volumetricHeatSource": 0.001
         }
-    }
+    },
+    "userDefinedFields":[]
```
Comment (yifan-flex) [inline tests/simulation/translator/ref/Flow360_CHT_three_cylinders.json:202]: Why?

File: flow360/component/simulation/framework/unique_list.py [hunk 2]
```diff
@@ -62,11 +62,14 @@ def _format_input_to_list(cls, input_data: Union[dict, list, Any]):
...
-            if "items" not in input_data:
+            # Add alias "stored_entities" so front end can monitor change in entities.
+            if "items" not in input_data and "stored_entities" not in input_data:
                 raise KeyError(
                     f"Invalid input to `entities` [UniqueItemList], dict {input_data} is missing the key 'items'."
                 )
-            return {"items": input_data["items"]}
+            if "items" in input_data:
+                return {"items": input_data["items"]}
+            return {"items": input_data["stored_entities"]}
```
Comment (yifan-flex) [inline flow360/component/simulation/framework/unique_list.py:72]: So UniqueItemList will accept EntityList data?

Comment (benflexcompute) [reply flow360/component/simulation/framework/unique_list.py]: Moved Point as entity

File: tests/simulation/translator/ref/Flow360_CHT_three_cylinders.json (hunk 1 above)
Comment (benflexcompute) [reply tests/simulation/translator/ref/Flow360_CHT_three_cylinders.json]: Removed. Turn out not necessary.

**Review by yifan-flex — APPROVED** [review]

**Pull request merged**